IdentityCore/src/webview/operations/MSIDWebResponseBrokerInstallOperation.m

```diff
@@ -67,7 +67,11 @@ - (nullable instancetype)initWithResponse:(__unused MSIDWebviewResponse *)respon
         
         return self;
     #else
-        *error = MSIDCreateError(MSIDErrorDomain, MSIDErrorInternal, @"Broker is not supported on Mac", nil, nil, nil, nil, nil, YES);
+        (void)response;
+        if (error)
+        {
+            *error = MSIDCreateError(MSIDErrorDomain, MSIDErrorInternal, @"Broker is not supported on Mac", nil, nil, nil, nil, nil, YES);
```

Comment:
this message is technically not correct anymore :) Authenticator installation link is not supported.